flake.nix

```diff
@@ -182,13 +178,6 @@
                     super.makeModulesClosure (x // {allowMissing = true;});
                 })
               ];
-            }
-            {
-              # Bootloader for RaspberryPi 4
-              boot.loader.raspberryPi = {
-                enable = true;
-                version = 4;
-              };
```

Comment:
I think these might not be possible to be removed -- the RPi boots from the GPU and might thus require its own bootloader to be present.